main.go

```diff
@@ -365,8 +366,9 @@ func run(state overseer.State) {
 				"detector", id,
 			)
 		}
-		// TODO: Add flag to ignore the default endpoint.
-		urls = append(urls, customizer.DefaultEndpoint())
+		if !*customVerifiersOnly {
+			urls = append(urls, customizer.DefaultEndpoint())
+		}
```

Comment:
Thoughts on appending the default if `len(urls) == 0`? Might be a better user experience if you want to overwrite one endpoint of only one of the possible detectors that can be configured.